packages/core/bootstrap/src/lib/server.ts

```diff
@@ -61,11 +62,20 @@ export const initHandler =
       const metricsMeta = METRICS_ENABLED
         ? { metricsMeta: { requestOrigin: getClientIp(req) } }
         : {}
+      const ip = req.ip
+      const hostname = req.hostname
       req.body.data = {
         ...(req.body.data || {}),
         ...toObjectWithNumbers(req.query),
         ...metricsMeta,
+        ip,
+        hostname,
       }
+      if (debug) {
```

Comment:
can we update this to more descriptive?  So maybe something like

`Received request from IP ${ip} and Host ${hostname}`